com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Added support for arm64 macOS.
 - The package depends on the Cloth Unity Module.
 - Fixed a bug, where degenerate triangles would create NaN normals.
+- Fixed a crash caused by empty geometry samples.
```

Comment:
Suggestion:
`Fixed the importer to prevent the Editor from crashing when importing meshes with empty geometry samples.`